dockertls/README.md

```diff
@@ -63,6 +63,34 @@ $env:DOCKER_TLS_VERIFY="1"
 docker version
 ```
 
+## Managing Multiple Hosts
+
+## First Host
+For the first host you must create an empty directory to hold the Certificate Authority Private Keys.
+Ideally the key and password should be kept separate and only provided when additional certificates are created.
+```
+mkdir $env:SystemDrive\DockerSSLCARoot
+mkdir $env:USERPROFILE\.docker
+docker run --rm `
+  -e SERVER_NAME=$(hostname) `
+  -e IP_ADDRESSES=127.0.0.1,192.168.254.135 `
```

Comment:
Adding all IP addresses (also the HNS nic) allowed me to run eg. portainer container like this

```powershell
$ip=(Get-NetIPAddress -AddressFamily IPv4 `
   | Where-Object -FilterScript { $_.InterfaceAlias -Eq "vEthernet (HNS Internal NIC)" } `
   ).IPAddress

docker kill portainer
docker rm -vf portainer

if (Test-Path $env:USERPROFILE\.docker\ca.pem) {
  docker run -d -p 8000:9000 `
    -v $env:USERPROFILE\.docker:C:\certs --name portainer portainer/portainer:windows `
    -H tcp://$($ip):2376 --tlsverify
} else {
  docker run -d -p 8000:9000 --name portainer portainer/portainer:windows -H tcp://$($ip):2375
}
```